mobile/lib/modules/asset_viewer/image_providers/cache/remote_image_cache_manager.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:flutter_cache_manager/flutter_cache_manager.dart';
+
+/// The cache manager for full size images [ImmichRemoteImageProvider]
+class RemoteImageCacheManager extends CacheManager with ImageCacheManager {
+  static const key = 'remoteImageCacheKey';
+  static final RemoteImageCacheManager _instance = RemoteImageCacheManager._();
+
+  factory RemoteImageCacheManager() {
+    return _instance;
+  }
+
+  RemoteImageCacheManager._()
+      : super(
+          Config(
+            key,
+            maxNrOfCacheObjects: 500,
```

Comment:
Can we limit the large remote image cache by total size instead of objects? Like 1GB etc.?

Anyhow, we can make this configurable in another PR

Reply:
No, this is only configurable by max number of cache objects, not by size.

https://github.com/Baseflow/flutter_cache_manager/issues/265